documents/policy/policy.tex

```diff
@@ -0,0 +1,162 @@
+\documentclass{scrreprt}
+\usepackage{listings}
+\usepackage{underscore}
+\usepackage[bookmarks=true]{hyperref}
+\usepackage[utf8]{inputenc}
+\usepackage[english]{babel}
+\usepackage{graphicx}
+\usepackage{xcolor}
+\usepackage{float}
+\hypersetup{
+    bookmarks=false,    % show bookmarks bar?
+    pdftitle={Team 2 Policies},    % title
+    pdfauthor={CMPT371 Team 2},                     % author
+    pdfsubject={Policy},                        % subject of the document
+    pdfkeywords={TeX, LaTeX, graphics, images, policy}, % list of keywords
+    colorlinks=true,       % false: boxed links; true: colored links
+    linkcolor=blue,       % color of internal links
+    citecolor=black,       % color of links to bibliography
+    filecolor=black,        % color of file links
+    urlcolor=purple,        % color of external links
+    linktoc=page            % only page is linked
+}%
+\def\myversion{1.0 }
+\date{}
+%\title{%
+
+%}
+\usepackage{hyperref}
+\begin{document}
+
+\graphicspath{ {images/} }
+
+\begin{flushright}
+    \rule{16cm}{5pt}\vskip1cm
+    \begin{bfseries}
+        \Huge{TEAM 2 POLICY}\\
+        \vspace{1.9cm}
+        for\\
+        \vspace{1.9cm}
+        CMPT371\\
+        \vspace{1.9cm}
+        \LARGE{Version \myversion approved}\\
+        \vspace{1.9cm}
+        Prepared by Team 2\\
+        \vspace{1.9cm}
+        University of Saskatchewan\\
+        \vspace{1.9cm}
+        \today\\
+    \end{bfseries}
+\end{flushright}
+
+\tableofcontents
+
+\chapter{Policy}
+
+\section{Members}
+
+\begin{table}[h!]
+\centering
+\begin{tabular}{ |p{2cm}||p{8cm}|  }
+    \hline
+    \multicolumn{2}{|c|}{Team Roles} \\
+    \hline
+    Member & Roles\\
+    \hline
+    Amanda & Project Lead\\
```

Comment:
Please review these to ensure all roles are correct. I guessed for a few of theme.

Reply:
Should be:
Evan - Project Lead
Eileen - Test Lead 
Amanda - Developer